com.eviware.soapui.metainfo.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>com.eviware.soapui</id>
+  <name>SoapUI Open Source</name>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>EUPL-1.1</project_license>
+  <developer_name>SmartBear Software</developer_name>
+  <summary>SoapUI OS is a functional testing solution for APIs and web services</summary>
+  <description>
+    <p>SoapUI Open Source alows developers and testers to explore and perform functional tests in REST, SOAP, and GraphQL APIs.</p>
+  </description>
+  <url type="homepage">https://www.soapui.org/</url>
+  <url type="bugtracker">https://github.com/SmartBear/soapui/issues</url>
+  <url type="faq">https://community.smartbear.com/t5/SoapUI-Open-Source/bd-p/SoapUI_OS</url>
+  <url type="help">https://www.soapui.org/docs/soap-and-wsdl/</url>
+  <url type="contact">https://smartbear.com/contact-us/</url>
+  <!-- not working yet https://github.com/flatpak/flatpak-builder/issues/480
```

Comment:
use the flaptak of flatpak-builder. `flatpak install org.flatpak.Builder`

some recent patches in appstream broke things and the flatpak contain all that is needed.